tests/test_recorders.py

```diff
@@ -1882,6 +1882,17 @@ class TestEventRecorder:
     """ Tests for EventRecorder """
     funcs = {"min": np.min, "max": np.max, "mean": np.mean, "median": np.median, "sum": np.sum}
 
+    @pytest.mark.parametrize("threshold_component", [StorageThresholdRecorder, StorageThresholdParameter])
+    def test_load(self, cyclical_storage_model, threshold_component):
+        """Test load method""" 
+        m = cyclical_storage_model
+        strg = m.nodes['Storage']
+        param = threshold_component(m, strg, 4.0, predicate='<=', name="trigger")
+        EventRecorder.load(m, {"name": "event_rec", "threshold": "trigger", "tracked_parameter": "inpt_flow"})
+        EventDurationRecorder.load(m, {"event_recorder": "event_rec", "recorder_agg_func": "max"})
```

Comment:
So, what happens  if you don't provide `recorder_agg_func` or `agg_func`? Does it error? Should we have a default here?

Reply:
yeah, it errors. We probably should have a default. Any preferences on what this should be?

Reply:
I think "mean" is the default for "agg_func" in general, so can likely be the same here. Make sure the docstring says the default value too.

Reply:
Done